tests/dist/transport/functions.cpp

```diff
@@ -62,6 +62,52 @@ int handlePointToPointFunction(
     return 0;
 }
 
+int handleManyPointToPointMsgFunction(
+  tests::DistTestExecutor* exec,
+  int threadPoolIdx,
+  int msgIdx,
+  std::shared_ptr<faabric::BatchExecuteRequest> req)
+{
+    faabric::Message& msg = req->mutable_messages()->at(msgIdx);
+
+    int groupId = msg.groupid();
+    uint8_t groupIdx = (uint8_t)msg.groupidx();
+    int numMsg = 10000;
+
+    // Set the broker to receive messages in order
+    auto& broker = faabric::transport::getPointToPointBroker();
+    broker.setMustOrderMessages(true);
```

Comment:
AFAICT this doesn't get set back to `false` anywhere. It should really do the following:

```cpp
bool originalMessageOrderingOn = broker.isMessageOrderingOn();
broker.setMessageOrderingOn(true);

...

broker.setMessageOrderingOn(originalMessageOrderingOn);
```